web/src/components/storage/utils.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
```

Comment:
NP: great having storage utils together in a separate files. However, I have no clear if it is better to place the file here, under `~/components/storage` or we should go for `~/utils` and move it to `~/utils/storage` and the current utils to `~/utils/common` or so. At the end, these utils can be used in no-component files too.

Anyway, just thinking aloud. I have no strong opinion in that regard at this moment, and what is better, we can do proposed change later.